apps/desktop/src/components/v3/StackTab.svelte

```diff
@@ -104,73 +101,64 @@
 			transform: translateY(-100%);
 			transition: transform var(--transition-fast);
 		}
-	}
-	.first {
-		border-radius: var(--radius-ml) 0 0 0;
-	}
 
-	.last {
-		border-right: none;
-	}
+		&.first {
+			border-radius: var(--radius-ml) 0 0 0;
+		}
 
-	.content {
-		display: flex;
-		align-items: center;
-		overflow: hidden;
-		position: relative;
-	}
+		&.last {
+			border-right: none;
+		}
 
-	.menu-button-wrap {
-		position: relative;
-		width: 0;
-	}
+		.content {
+			min-width: 75px;
+			display: flex;
+			align-items: center;
+			overflow: hidden;
+		}
 
-	.tab:hover .name,
-	.menu-open .name {
-		/* Shrinks name to make room for hover button. */
-		width: calc(100% - 32px);
-	}
+		&:active .menu-btn,
+		&:hover .menu-btn,
+		&:focus-within .menu-btn {
+			opacity: 1;
+			min-width: 32px;
+			padding: 8px;
+		}
 
-	.tab:hover .menu-button-wrap,
-	.menu-open .menu-button-wrap {
-		opacity: 1;
-		width: 32px;
-		/* We want the container to not take up extra space. */
-		margin-left: -32px;
-		/* But still be visible where it would normally display. */
-		transform: translateX(32px);
-	}
+		.menu-btn {
+			position: relative;
+			opacity: 0;
+			width: 0;
+			flex-shrink: 0;
 
-	.menu-button {
-		display: flex;
-		align-items: center;
-		justify-content: center;
-		color: var(--clr-text-2);
-		padding: 8px;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			color: var(--clr-text-2);
 
-		&.menu-open,
-		&:hover {
-			color: var(--clr-text-1);
+			&:hover {
+				color: var(--clr-text-1);
+			}
 		}
-	}
 
-	.tab:not(.selected):hover,
-	.tab:not(.selected):focus-within {
-		background: var(--clr-stack-tab-inactive-hover);
-	}
+		&:not(.selected):hover,
+		&:not(.selected):focus-within {
+			background: var(--clr-stack-tab-inactive-hover);
+		}
 
-	.tab:not(.selected):focus-within {
-		&::after {
-			transform: translateY(0);
-			background: var(--clr-border-1);
+		&:not(.selected):focus-within {
+			&::after {
+				transform: translateY(0);
+				background: var(--clr-border-1);
+			}
 		}
-	}
 
-	.selected {
-		&::after {
-			transform: translateY(0);
-			background: var(--clr-theme-pop-element);
-			z-index: var(--z-ground);
+		&.selected {
+			&::after {
+				transform: translateY(0);
+				background: var(--clr-theme-pop-element);
+				z-index: var(--z-ground);
+			}
```

Comment:
I removed the css nesting for all these classes just yesterday since it doesn't really serve a useful purpose. How come you're adding it back in?